src/main/java/io/github/thebusybiscuit/slimefun4/api/MinecraftVersion.java

```diff
@@ -193,4 +271,14 @@ public boolean isBefore(@Nonnull MinecraftVersion version) {
         return version.ordinal() > this.ordinal();
     }
 
+    /**
+     * Checks whether this {@link MinecraftVersion} is older than the specified minecraft and patch versions
+     * @param minecraftVersion The minecraft version
+     * @param patchVersion The patch version
+     * @return True if this version is before, False if this version is virtual or otherwise.
+     */
+    public boolean isBefore(int minecraftVersion, int patchVersion) {
+        return !isVirtual() && (this.majorVersion < minecraftVersion && this.minorVersion < patchVersion);
```

Comment:
If for 1.21 we have 1_21, wouldn't calling 1_20_5#isBefore(21, 0) fail because the minor version 5 is larger than -1? And if so would we need to adjust it to be `this.majorVersion < minecraftVersion || this.minorVersion < patchVersion` or am I missing something